cmd/result-aggregator/main.go

```diff
@@ -55,6 +57,7 @@ func gatherOptions() (options, error) {
 	fs.DurationVar(&o.gracePeriod, "gracePeriod", time.Second*10, "Grace period for server shutdown")
 	fs.StringVar(&o.username, "username", "", "Username to trust for clients.")
 	fs.StringVar(&o.password, "password-file", "", "File holding the password for clients.")
+	fs.StringVar(&o.htpasswdFile, "htpasswd-file", "", "Authenticate against a htpasswd file. Entries must be created with \"htpasswd -p\" for no encryption.")
```

Comment:
Nit: This is not encryption, this is hashing

Reply:
I know. 

I copied the code over from (removing the hashing/encription part)
https://github.com/openshift/oauth-proxy/blob/master/htpasswd.go

https://github.com/openshift/oauth-proxy/blob/4bd4705011c19a7556551c9a1dfaaa5eacc7898c/main.go#L64
```
	flagSet.String("htpasswd-file", "", "additionally authenticate against a htpasswd file. Entries must be created with \"htpasswd -s\" for SHA encryption")
```

I feel the encryption is not needed since the user/password is generated (the original way is plain-text too). So

> Entries must be created with \"htpasswd -p\" for no encryption."

```bash
$ htpasswd -bnp username password
Warning: storing passwords as plain text might just not work on this platform.
username:password
```

my bad choice again?

Reply:
I guess that is fine, but then do not name this `htpasswd`, because its not

Reply:
Will rename.

Reply:
Updated.